docker-compose.sample.yml

```diff
@@ -14,35 +14,37 @@ networks:
     external: false
 
 services:
+ 
   traefik:
-    image: traefik:1.7-alpine
+    image: "traefik:v2.2.11"
+    container_name: "traefik"
```

Comment:
I'll remove them, since the consensus was to remove them, however I personally like using them because a) this way it does not matter what the folder name where docker-compose is and b) the names are shorted and easier to read.

If you follow `readme.md` it puts `docker-compose`.yml in `/mnt/docker` and the names will be `docker_mariadb_1` and not `mailserver_mariadb_1` unless you specify `-p` for `docker-compose`.